cg/meta/archive/archive.py

```diff
@@ -103,12 +103,21 @@ def archive_all_non_archived_spring_files(
                 archive_location=archive_location,
             )
 
-    def retrieve_samples(self, sample_internal_ids: list[str]) -> None:
-        """Retrieves the archived spring files for a list of samples."""
-        samples: list[Sample] = [
-            self.status_db.get_sample_by_internal_id(sample_internal_id)
-            for sample_internal_id in sample_internal_ids
+    def retrieve_case(self, case_id: str):
+        case = self.status_db.get_case_by_internal_id(case_id)
```

Comment:
Add type hint

Reply:
What does this function do? It says retrieve, but dies not seem to return anything?

Reply:
It submits a job to retrieve files from the data lake, and updates Housekeeper with the retrieval job id. Added it to the doc string now. Would you prefer a name change of the method as well @henrikstranneheim ?

Reply:
The doc str made it much clearer